addon/styles/x-tree.css

```diff
@@ -1,18 +1,18 @@
-.tree-branch {
+.x-tree-branch {
```

Comment:
Any benefit in changing these? I'd prefer leaving them they way they are for compatibility (css changes are harder to track down).

I'm thinking in a near future it would be nice to start leveraging angular components and drop the `x-`:
`<Tree @model={{nodes}} />`

Reply:
No. Did a search/replace for light-tree => x-tree. Let me see if I can fix these.